Bridge zombienet tests - move all test scripts to the same folder

serban300 opened the pull request:
Related to https://github.com/paritytech/polkadot-sdk/issues/3242

bkontur commented on bridges/zombienet/environments/rococo-westend/bridges_rococo_westend.sh:
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```
@serban300 we reference this script in `polkadot-sdk/cumulus/parachains/runtimes/bridge-hubs/README.md` e.g. `./cumulus/scripts/bridges_rococo_westend.sh run-relay`, could you please also fix/check that README.md that it is still working?

bkontur commented:
I just thinking about the directory structure how actually looks like:
```
zombienet
- environments/rococo-westend
- helpers
- scripts
- tests
- utils
- run-tests.sh
- run-new-test.sh
```

so my thought:
- `scripts` and `utils` can be join together, they both contains bash utility scripts
- `helpers` they are also kind of scripts/utils, but contains just `js-helpers` actually
- maybe all this `helpers` / `scripts` / `utils` could be join and released as a `.tar` so that we could reuse that for `polkadot-fellows` or something like that
- `zombienet` what about renaming to `testing` (maybe we will change zombienet for something else in the future)? Also some stuff from `utils` is not related to the zombienet

serban300 commented:
@bkontur addressed your comments. PTAL !

> * `scripts` and `utils` can be join together, they both contains bash utility scripts

Yes, I agree they both contain bash scripts, but `scripts` will be deleted after we migrate test `0003` to the new approach. So I would leave them as they are right now, and then just directly deleted them.

> * `helpers` they are also kind of scripts/utils, but contains just `js-helpers` actually

Renamed to `js-helpers`

> * maybe all this `helpers` / `scripts` / `utils` could be join and released as a `.tar` so that we could reuse that for `polkadot-fellows` or something like that

Yes, that's also what I'm thinking. Would be good to have some way of packaging these together and reuse them in other repos if needed. Not sure exactly how yet.



> * `zombienet` what about renaming to `testing` (maybe we will change zombienet for something else in the future)?

Yes, makes sense. Done
